apps/backend/prisma/client/sql/countDistinctRecipientsInTimeRange.d.ts

```
@@ -1,22 +1,15 @@
  * @param timestamp
  * @param timestamp
  */
-export const countDistinctRecipientsInTimeRange: (
-  text: string,
-  timestamp: Date,
-  timestamp: Date,
-) => $runtime.TypedSql<
-  countDistinctRecipientsInTimeRange.Parameters,
-  countDistinctRecipientsInTimeRange.Result
->;
+export const countDistinctRecipientsInTimeRange: (text: string, timestamp: Date, timestamp: Date) => $runtime.TypedSql<countDistinctRecipientsInTimeRange.Parameters, countDistinctRecipientsInTimeRange.Result>
```

Biome (1.9.4)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)